pkg/lightning/config/config.go

```diff
@@ -535,13 +536,20 @@ func (cfg *Config) Adjust(ctx context.Context) error {
 		return errors.New("invalid config: `mydumper.csv.separator` and `mydumper.csv.delimiter` must not be prefix of each other")
 	}
 
+	if len(csv.Terminator) > 0 && cfg.Mydumper.StrictFormat {
```

Comment:
@glorv Terminator and StrictFormat are currently mutually exclusive because the latter is hard-coded to use `[\r\n]` for now. There's no problem using TrimLastSep and Terminator together.

Reply:
But this will trim the last two seps, is there any scenario that actually need this? BTW, do we want to finally replace `TrimLastSep` with `Terminator` or they just two different feature?

Reply:
the original purpose of TrimLastSep was to support importing [TPC-H generated data](https://docs.pingcap.com/tidb/stable/migrate-from-csv-using-tidb-lightning#tpc-h-dbgen) in the form

```
a|b|c|
d|e|f|
```

, which we can use `terminator = "|\n"` instead. 

i don't know if there are any other use cases 😅